experimental/record-tuple.md

```diff
@@ -0,0 +1,28 @@
+# [Record & Tuple][proposal-record-tuple]
+
+## Expressions
+
+### RecordExpression
+
+```js
+interface RecordExpression <: Expression {
+    type: "RecordExpression";
+    properties: [ Property | SpreadElement ];
```

Comment:
Hmm it might be a bit early to go into details, so I suppose we could go as-is for now, but my understanding is that records will be more limited (e.g. no getters/setters/methods, right?). If so, we might want to have a separate RecordProperty node down the road.

Reply:
Good catch. In Babel we have `ObjectMethod` and `ObjectProperty` node, I just realize that they are both `Property` in estree. I am good to introduce a `RecordProperty`, will update this PR later.